django_pam/accounts/views.py

```diff
@@ -147,7 +147,7 @@ def get_context_data(self, **context):
         :rtype: dict
         """
         context[self.redirect_field_name] = self.get_success_url()
-        return context
+        return super(LoginView, self).get_context_data(**context)
 
```

Comment:
If I remember correctly, I purposely didn't want the super, because it puts the `form` object in the context which caused an issue with AJAX requests. You only changed a few lines, so I will merge your fixes in then run my tests and see what happens. You should be able to run the tests also if you want `make coverage` does things in the correct order. Thanks for the fixes, I was going to find this issue in the next month or so anyway as I'll be needing this package for an SSO project I'm working on for Cisco.